types/src/model/wallet.rs

```diff
@@ -421,6 +425,23 @@ pub struct ListReceivedByAddressItem {
     pub txids: Vec<Txid>,
 }
 
+/// Models the result of JSON-RPC method `listreceivedbylabel`.
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct ListReceivedByLabel(pub Vec<ListReceivedByLabelItem>);
+
+/// Item returned as part of `listreceivedbylabel`.
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct ListReceivedByLabelItem {
+    /// Only returned if imported addresses were involved in transaction.
+    pub involves_watchonly: Option<bool>,
```

Comment:
```suggestion
    pub involves_watch_only: Option<bool>,
```

Reply:
I was unsure of this but we have one instance already: `is_watch_only`. If you feel strongly you could change them both.

Reply:
Yeah we shouldn't follow too closely what Core does re word splitting, we have a totally different philosophy here.